docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,77 @@
+# How To Contribute
+
+## Some Ways to Contribute
+- Getting started with [L3AF](https://wiki.lfnetworking.org/display/L3AF/Getting+Started+with+L3AF)
+- Report potential bugs
+- Suggest product enhancements
+- Increase our test coverage
+- Fix a bug
+- Implement a requested enhancement
+- Improve our guides and documentation
+- Contribute code
+
+### Report potential bugs
+
+First, **ensure the bug was not already reported** by searching on GitHub under
+[Issues](https://github.com/l3af-project/l3afd/issues).
+
+If you did not find a related bug, you can help us by
+[submitting a GitHub Issue](https://github.com/l3af-project/l3afd/issues/new).
+A good bug report provides a detailed description of the issue and step-by-step instructions
+for reliably reproducing the issue.
+
+We will aim to triage issues in weekly TSC meetings. In case we are unable to repro the issue,
```

Comment:
```suggestion
We will aim to triage issues in [weekly TSC meetings](https://wiki.lfnetworking.org/display/L3AF/Community+Meetings). In case we are unable to repro the issue,
```